easybuild/easyconfigs/b/Bio-EUtilities/Bio-EUtilities-1.76-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,88 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Bundle'
+
+name = 'Bio-EUtilities'
+version = '1.76'
+
+homepage = 'https://github.com/bioperl/bio-eutilities'
+description = "BioPerl low-level API for retrieving and storing data from NCBI eUtils"
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+builddependencies = [('binutils', '2.32')]
+dependencies = [
+    ('Perl', '5.30.0'),
+    ('BioPerl', '1.7.2'),
+]
+
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perldoc -lm %(ext_name)s ", "")
+
+exts_list = [
+    ('ExtUtils::MakeMaker', '7.44', {
+        'source_tmpl': 'ExtUtils-MakeMaker-7.44.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BI/BINGOS'],
+        'checksums': ['52a18f8271250faf5f3527499dd2b78d3b4fd3b064408d0cfdda9a3538887188'],
+    }),
+    ('IO::Handle', '1.42', {
+        'source_tmpl': 'IO-1.42.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TO/TODDR'],
+        'checksums': ['eec5cc33a6cddba8b5d2425b60752882add7e4d41b7431a0ea4dcd73cc1f8cca'],
+    }),
+    ('Test::More', '0', {
+        'source_tmpl': 'Test-Simple-1.302171.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/E/EX/EXODIST'],
+        'checksums': ['e27f90d2b2a6bc6ffa7675a072c2f41d5caffd99858dc69b2030940cc138368a'],
+    }),
+    ('Bio::ASN1::EntrezGene', '1.73', {
+        'source_tmpl': 'Bio-ASN1-EntrezGene-1.73.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/C/CJ/CJFIELDS'],
+        'checksums': ['f9e778db705ce5c35ad2798e38a8490b644edfdc14253aa1b74a1f5e79fc6a4b'],
+    }),
+    ('Cwd', '3.75', {
+        'source_tmpl': 'PathTools-3.75.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/X/XS/XSAWYERX'],
+        'checksums': ['a558503aa6b1f8c727c0073339081a77888606aa701ada1ad62dd9d8c3f945a2'],
+    }),
+    ('Data::Dumper', '2.173', {
+        'source_tmpl': 'Data-Dumper-2.173.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/X/XS/XSAWYERX'],
+        'checksums': ['697608b39330988e519131be667ff47168aaaaf99f06bd2095d5b46ad05d76fa'],
+    }),
+    ('LWP::UserAgent', '6.06', {
+        'source_tmpl': 'libwww-perl-6.43.tar.gz',
```

Comment:
different version, probably need to use `6.43` above?

Reply:
Same here: https://metacpan.org/pod/LWP::UserAgent
has dl link: https://cpan.metacpan.org/authors/id/O/OA/OALDERS/libwww-perl-6.44.tar.gz

Reply:
But the version seems to be incorrect indeed :)